drivers/drivers.go

```diff
@@ -0,0 +1,285 @@
+// Package drivers abstracts different object storages, such as local, s3
```

Comment:
It was a little hard to review the changes here, since I can't tell what you copied from `go-livepeer` and what changes you made after that.. Would've been helpful to separate the copying on a separate commit so the review could be done only on the changes after that.

In the current state, my flow was to just review the second commit which made some changes here, then reviewed the `fs.go` driver, since the first commit message mentioned adding file system support. Is there anything else I should pay attention to?

Reply:
I think removal of `glog` is the only change to existing logic worth noting.